.cspell.json

```diff
@@ -9,7 +9,8 @@
     ],
     "allowCompoundWords": true,
     "ignorePaths": [
-        "*.exe"
+        "*.exe",
+        "edk2toolext/tests/test_edk2_logging.py"
```

Comment:
Please change this to all tests. I'm not particularly worried about spelling in tests and I'd like to keep it consistent.